test/fixtures/valid-config.json

```diff
@@ -36,7 +36,14 @@
         "scopes": ["scope"],
         "landing_page": "page",
         "cryptor_secret": "some-secret",
-        "cookie_name_prefix": "my-app"
+        "cookie_name_prefix": "my-app",
+        "id_token": {
+          "preamble": "Bearer",
+          "header": "authorization"
+        },
+        "access_token": {
```

Comment:
@nickrmc83 Do we understand correctly that this PR should be safe to merge as long as the user never configures "access_token" in their json config file?  We think that bug with setting multiple cookies will always actually set whichever `set-cookie` header was added last to the list of headers in the `check` response.  Since the ID Token header is added last, and the ID Token cookie is the only required cookie in the error checking code, it seems like it would work.

For the docs, based on your current PR code, we're thinking that we would document that the `id_token.header` is required and the `id_token.preamble`, while not technically required, should probably always be set to `Bearer` in practice. Does that sound right?  If `id_token.header` is required, than perhaps we should make it an error when the user fails to supply it?  Or would it be preferred to make both optional and make them default to `authorization` and `Bearer` when they are not supplied?

Reply:
c2c026409ed98feceda56a8ede73b92a7e9e600b includes validation and specification of required fields leveraging protobuf annotations.